apps/web/next.config.js

```diff
@@ -243,64 +251,70 @@ const nextConfig = {
         source: "/org/:slug",
         destination: "/team/:slug",
       },
-      {
-        source: "/:user/avatar.png",
-        destination: "/api/user/avatar?username=:user",
-      },
       {
         source: "/team/:teamname/avatar.png",
         destination: "/api/user/avatar?teamname=:teamname",
       },
-      {
-        source: "/forms/:formQuery*",
-        destination: "/apps/routing-forms/routing-link/:formQuery*",
-      },
-      {
-        source: "/router",
-        destination: "/apps/routing-forms/router",
-      },
-      {
-        source: "/success/:path*",
-        has: [
-          {
-            type: "query",
-            key: "uid",
-            value: "(?<uid>.*)",
-          },
-        ],
-        destination: "/booking/:uid/:path*",
-      },
-      {
-        source: "/cancel/:path*",
-        destination: "/booking/:path*",
-      },
+
+      // When updating this also update pagesAndRewritePaths.js
+      ...[
+        {
```

Comment:
Clubbed rewrites in an array that conflict with either /:user or /:user/:type. 
Such rewrites should be ensured to be excluded in the org rewrites as org rewrites happen in `beforeFiles` which executes even before dynamic routes.